src/components/DeleteHelicopter.jsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  fetchHelicopters,
+  deleteHelicopter,
+  updateHelicopters,
+} from "../features/helicopters/helicopterSlice";
+import LayoutComponent from "../Layout";
+import { Button, Table, Popconfirm } from 'antd';
+import { QuestionCircleOutlined } from "@ant-design/icons";
+
+const DeleteHelicopterComponent = () => {
```

Comment:
- [ ] I can't delete helicopters, or reservations:

![image](https://github.com/emmanuelsteven/heli-hire-captsone-frontend/assets/57421823/e7dd71d3-1adb-4805-b489-3a10350c573a)

Kindly, make it so helicopters can be deleted. 👍 

_Tested on Chrome and Firefox_

Reply:
fixed